apps/web/.eslintrc.js

```diff
@@ -36,7 +36,8 @@ module.exports = {
       },
     ],
     "react-hooks/rules-of-hooks": 'error',
-    "react-hooks/exhaustive-deps": 'warn'
+    "react-hooks/exhaustive-deps": 'warn',
+    "import/extensions": 'off',
```

Comment:
Would love an alternative suggestion if anyone has one! This rule failed repeatedly in CI for the Panda `styled-system` imports, but none of the tweaks I made seemed to deactivate the rule. ([example commit](https://github.com/novuhq/novu/pull/5283/commits/a8dcdb1aab805c99ea3be804bdc7d6a1d02a6d6f))